### PR TITLE
Added sql command

### PR DESCRIPTION
**Pull request by eshleebien**
added command

### CODE REVIEW

File: commands/sql_query.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            app.log.info('Please wait ...');
+
+            mysql.open(app.config.get('database').heartbeat)
+                .query(query, process_rows);
+        },
+
+        process_rows = function (err, rows) {
+            if (err) {
+                mysql.end();
+                app.log.error(err);
```
Comment (robertsambuena) [inline commands/sql_query.js]: new line

File: commands/sql_query.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+module.exports = function (file, cb) {
+
+    if (typeof cb === 'string') {
+        this.log.info('Invalid usage');
+
+        return;
+    }
+
+    var query = '',
+        app = this,
```
Comment (robertsambuena) [inline commands/sql_query.js]: new line

**Comment by robertsambuena** [comment]
:up:

**Comment by Jhoem** [comment]
:up:

**Pull request merged**